app/ui-utils/client/lib/readMessages.js

```diff
@@ -95,8 +95,9 @@ export const readMessage = new (class extends Emitter {
 		this.enabled = document.hasFocus();
 	}
 
-	isEnable() {
-		return this.enabled === true;
+	isEnabled() {
+		this.enable();
```

Comment:
this doesn't make much sense

Reply:
I know, but this is +- how we are doing on the rest of code, so I just kept it at least for know